main.py

```diff
@@ -298,6 +299,13 @@ async def main(args):
   parser.add_argument("--bandwidth", default=1000, help="Bandwidth limit per IP, in kilobytes per second.")
   parser.add_argument("--connections", default=30, help="Connections limit per IP, in kilobytes per second.")
   parser.add_argument("--window", default=60, help="Fixed window length for rate limits, in seconds.")
+  parser.add_argument("--config", default="config.json", help="The config file location.")
```

Comment:
Also, don't make this try to load a nonexistent file as a default. It should only a attempt to load a config file if the `--config` argument is passed in.